canton/BUILD.bazel

```diff
@@ -113,3 +106,16 @@ cp **/lib/*.jar $@
     ],
     visibility = ["//visibility:public"],
 )
+
+java_import(
+    name = "lib-ee",
+    jars = [":canton-lib-ee.jar"],
+    tags = ["canton-ee"],
+)
+
+java_binary(
```

Comment:
Should this have the `canton-ee` tag, or will it be skipped automatically if the tag is disabled because it depends on a tagged target?